tests/fuzz/runFuzzTest.sh

```diff
@@ -54,6 +54,11 @@ else
     export LD_PRELOAD="$GLOBAL_OVERRIDES"
 fi
 
+if [[ ! -z "$AFL_FUZZ" ]]; then
+    exit 0
```

Comment:
Couldn't we run `afl-fuzz` here instead?

Reply:
Since it currently isn't integrated with our CI system I didn't want to add pointless extra run time to our fuzzing if it was enabled before integration, but I'm willing to set it up regardless since that's unlikely.

Reply:
I think it would be nice to have so the interface for running fuzz tests locally stays the same (with the additional environment variable AFL_FUZZ)

Reply:
It should be safe to add it here- without the `AFL_FUZZ`, it won't impact current runs.

Reply:
Added! Enabling AFL_FUZZ should run AFL tests, otherwise it'll operate as normal.